torch/onnx/_internal/fx/registration.py

```diff
@@ -192,83 +80,136 @@ def _initiate_registry_from_torchlib(
 
+        Args:
+            domain: The domain of the operator to get.
+            op_name: The name of the operator to get.
+            overload: The overload of the operator to get. If it's default overload,
+                leave it to None.
         Returns:
-            The _SymbolicFunctionGroup object corresponding to the given name, or None if the name is not in the registry.
+            A list of SymbolicFunctions corresponding to the given name, or None if
+            the name is not in the registry.
         """
-        return self._registry.get(name)
+        internal_name = OpName.from_string(
+            domain=domain, op_name=op_name, overload=overload
+        ).internal_name()
+        if (functions := self._registry.get(internal_name)) is not None:
+            return functions
+        return None
 
     @_beartype.beartype
-    def is_registered_op(self, name: str, version: int) -> bool:
-        """Returns whether the given op is registered for the given opset version.
+    def is_registered_op(
+        self, domain: str, op_name: str, overload: Optional[str] = None
+    ) -> bool:
+        """Returns whether the given op is registered: torch.ops.<domain>.<op_name>.<overload>.
 
         Args:
-            name: The qualified name of the function to check.
-            version: The opset version to check.
+            domain: The domain of the operator to check.
+            op_name: The name of the operator to check.
+            overload: The overload of the operator to check. If it's default overload,
+                leave it to None.
 
         Returns:
-            True if the given op is registered for the given opset version, otherwise False.
+            True if the given op is registered, otherwise False.
         """
-        functions = self.get_function_group(name)
-        if functions is None:
-            return False
-        return functions.get(version) is not None
+        functions = self.get_functions(
+            domain=domain, op_name=op_name, overload=overload
+        )
+        return functions is not None
 
     @_beartype.beartype
-    def all_functions(self) -> Set[str]:
+    def _all_registered_ops(self) -> Set[str]:
         """Returns the set of all registered function names."""
         return set(self._registry)
+
+
+@dataclasses.dataclass(frozen=True)
+class OpName:
+    """A class representing an operator name in internal ONNX converter."""
+
+    domain: str
+    op_name: str
+    overload: str
+
+    @classmethod
+    @_beartype.beartype
+    def from_string(
+        cls, domain: str, op_name: str, overload: Optional[str] = None
+    ) -> OpName:
+        # NOTE: in PyTorch, the overload could be unprovided to indicate the
+        # default overload
+        if overload is None or overload == "":
+            overload = "default"
+        return cls(domain, op_name, overload)
+
+    @classmethod
+    @_beartype.beartype
+    def from_full_name(cls, full_name: str) -> OpName:
```

Comment:
Define what a "full_name" here in docstring?

Reply:
Done

Reply:
From code "full_name" is rather `<namespace>::<op_name>[.<overload>]`

Reply:
Done